dev-env/bin/dad-add-source-dep.sh

```diff
@@ -1,9 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
This file has not been touched since open-sourcing and is not referenced anywhere. I think I'd rather delete it than update it, but I guess there's no harm.

Reply:
Thanks for the context. No need to flog a dead horse - I've reverted the change to this file.